crates/gpui/src/text_system/line_wrapper.rs

```diff
@@ -195,6 +196,46 @@ impl LineWrapper {
     }
 }
 
+//----------------------------------------------
```

Comment:
Thanks for making these into a test! Can you remove this comment now that they're in the test?

Reply:
Sure! Done.